deploy/static/static.yaml

```diff
@@ -21,18 +21,24 @@ spec:
     spec:
       containers:
         - name: from-file-prometheus-exporter
-          image: python:alpine3.9
+          image: golang:alpine
           env:
             - name: METRICS_FILE_URL
-              value: "<ADD_YOUR_FILE_URL_HERE>" # You can use private gist url.
+              value: "" # You can use private gist url.
+            - name: TESTER_SCRIPT_URL
+              value: "https://raw.githubusercontent.com/newrelic/nri-prometheus/santalla/mockexporter/internal/mockexporter/loadtest.go".
           ports:
             - name: metrics
               containerPort: 8080
           command: ["/bin/sh","-c"]
           # The reason of using a URL instead of a config map is that the latest has a limitation of up to 1MB
-          args: ["wget $METRICS_FILE_URL -O /etc/from-file-prometheus-exporter/metrics; python -m http.server -b 0.0.0.0 -d /etc/from-file-prometheus-exporter/ 8080"]
+          args:
+            - |
+              [[ -n "$METRICS_FILE_URL" ]] && wget $METRICS_FILE_URL -O /app/metrics
+              [[ -n "$TESTER_SCRIPT_URL" ]] && wget $TESTER_SCRIPT_URL -O /app/loadtest.go
+              go run /app/loadtest.go -addr :8080 -metrics /app/metrics -latency 50 -latency-variation 15
```

Comment:
not sure if this is going to work outside the gopath without go.mod file.

Reply:
It does (at least locally) as a kind of an edge case, because all dependencies are from the standard library. It will probably be messier if it had to download some.